helidon-ls-ext/helidon.jdt.ext/README.md

```diff
@@ -0,0 +1,15 @@
+This project extends the JDT.LS by providing an extension point `org.eclipse.lsp4mp.jdt.core.propertiesProviders` for `PropertiesManager`. 
```

Comment:
Why the `dot` separated name ? Is this required by Tycho?

Reply:
fixed